config_test.go

```diff
@@ -28,6 +31,42 @@ packages:
 	assert.Equal(t, pkg, Package{Repo: "github.com/grpc/grpc-go"})
 }
 
+func TestParseGodocServer(t *testing.T) {
+	tests := []struct {
+		give string
+		want string
+	}{
+		{"example.com", "example.com"},
+		{"example.com/", "example.com"},
+		{"http://example.com/", "example.com"},
+		{"https://example.com/", "example.com"},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.give, func(t *testing.T) {
+			path, clean := TempFile(t, fmt.Sprintf(`
+godoc:
+  host: %q
+url: google.golang.org
+packages:
+  grpc:
+    repo: github.com/grpc/grpc-go
+`, tt.give))
+			defer clean()
+
+			config, err := Parse(path)
+			require.NoError(t, err)
+
+			assert.Equal(t, tt.want, config.Godoc.Host)
+			assert.Equal(t, "google.golang.org", config.URL)
+
+			pkg, ok := config.Packages["grpc"]
+			assert.True(t, ok)
```

Comment:
nit: this should be a require since the next assert doesn't make sense without this step